### PR TITLE
GetShard: support multiple chunks in the same item

### PR DESCRIPTION
**Pull request by alexbondar92**

### CODE REVIEW

File: pkg/dataplane/test/sync_test.go [hunk 1]
```diff
@@ -1367,22 +1367,26 @@ func (suite *syncStreamBackupRestoreTestSuite) TestStream() {
...
-			streamBackup[shardName].Chunks[chunkId] = &Chunk{}
-		}
-		if chunkMetadata != nil {
-			(*streamBackup[shardName].Chunks[chunkId]).ChunkMetadata = chunkMetadata
-		}
-		if len(chunkDataArray) != 0 {
-			(*streamBackup[shardName].Chunks[chunkId]).Data = chunkDataArray
+		for chunkId := range chunkMap {
+			if _, ok := streamBackup[shardName].Chunks[chunkId]; !ok {
+				streamBackup[shardName].Chunks[chunkId] = &Chunk{}
```
Comment (FelGel) [inline pkg/dataplane/test/sync_test.go:1379]: `chunkMap` is already map of chunks by `chunkID`

File: pkg/dataplane/item.go [hunk 2]
```diff
@@ -85,55 +85,61 @@ func (i Item) GetFieldUint64(name string) (uint64, error) {
...
+func (i Item) GetShard() (map[int]*ItemChunk, *ItemCurrentChunkMetadata, error) {
 	const streamDataPrefix = "__data_stream["
 	const streamMetadataPrefix = "__data_stream_metadata["
 	const offsetPrefix = "__data_stream[0000]["
 
-	var chunkDataArray []*ItemChunkData
-	chunkMetaData := ItemChunkMetadata{}
 	currentChunkMetadata := ItemCurrentChunkMetadata{}
-	var chunkID int
+	chunkMap := make(map[int]*ItemChunk)
```
Comment (FelGel) [inline pkg/dataplane/item.go:94]: if `chunkMap` contains multiple entries in what order (if it matters) should the chunks be restored?

Comment (alexbondar92) [reply pkg/dataplane/item.go:94]: I don't commit any spacial order in the map,
this pr not changing the expected restore order.

File: pkg/dataplane/test/sync_test.go (hunk 1 above)
Comment (alexbondar92) [reply pkg/dataplane/test/sync_test.go:1379]: you right, but there could be a scenario that there is chunks that is scattered over multiple Items...

Comment (FelGel) [reply pkg/dataplane/test/sync_test.go:1379]: what I meant is - you have two places where you define the same `Chunk` struct

Comment (alexbondar92) [reply pkg/dataplane/test/sync_test.go:1379]: point me to which ones?

Comment (FelGel) [reply pkg/dataplane/test/sync_test.go:1379]: https://github.com/v3io/v3io-go/blob/9e9b44868989d454367ad3e0c6c6cf8e55009e24/pkg/dataplane/types.go#L473 
and 
https://github.com/v3io/v3io-go/blob/9e9b44868989d454367ad3e0c6c6cf8e55009e24/pkg/dataplane/test/sync_test.go#L1358

Comment (alexbondar92) [reply pkg/dataplane/test/sync_test.go:1379]: you are right, change to the same struct

**Review by FelGel — APPROVED** [review]

**Pull request merged**